ens/main.py

```diff
@@ -271,7 +280,16 @@ def resolve(self, name: str, get: str = 'addr') -> Optional[Union[ChecksumAddres
         else:
             return None
 
-    def resolver(self, normal_name: str) -> Optional['Contract']:
+    def resolver(self, normal_name: str, name: str = None) -> Optional['Contract']:
```

Comment:
What do you think about setting `normal_name = None` here as well, and then the interim API that satisfies the warning looks like: `resolver(name='test.eth')` instead of `resolver('test.eth', name='test.eth')`? We'd probably want to do some validation too, to make sure both aren't passed in.

Reply:
Yeah, I like that way better. I also forgot about this PR before last release... I'll clean this up and run it by for another review 👌

Reply:
@kclowes this should be ready to review again. lmk if the latest commit is good here 👀